BonusCalcListener/UseCase/PayElementMapper.cs

```diff
@@ -34,6 +31,18 @@ public PayElement BuildPayElement(WorkOrderOperativeSmvData eventData)
             }
         }
 
+        private static bool EventIsOvertime(WorkOrderOperativeSmvData eventData)
+        {
+            if (eventData.PaymentType == null)
+            {
+                // old check
+                return eventData.IsOvertime.HasValue && (bool) eventData.IsOvertime;
+            }
+
+            // new check using PaymentType enum
+            return eventData.PaymentType == Domain.PaymentType.Overtime;
```

Comment:
Do you need the `Domain.PaymentType.Overtime` here or will just `PaymentType.Overtime` work since you've imported the `BonusCalcListener.Domain` ?